doc/source/notebooks/theory/Sanity_check.pct.py

```diff
@@ -101,13 +102,16 @@
 # %%
 for m in models:
     opt = gpflow.optimizers.Scipy()
-    if isinstance(m, gpflow.models.SVGP):
-        loss_fn = lambda: -m.log_marginal_likelihood(data)
+    if isinstance(m, gpflow.models.ExternalDataTrainingLossMixin):
```

Comment:
can we make a helper function for this utility?

Reply:
there already is ! should've used it. good catch

Reply:
there wasn't, just a similar one, but i've added one now